### PR TITLE
3671 admin authz accessid

### PR DESCRIPTION
**Pull request by utle**
admin authz to support user and group accessId

### CODE REVIEW

**Comment by utle** [comment]
fixed #3671

**Comment by utle** [comment]
#build
#spawn.fullfat.buckets=com.ibm.ws.management.security.authorization_fat

**Comment by utle** [comment]
#build

File: dev/com.ibm.ws.management.security/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -21,5 +21,13 @@ role.administrator.desc=A collection of users and/or groups assigned the server
 role.user=User name
 role.user.desc=User assigned a role.
 
+role.userAccessId=User access ID
+role.userAccessId.desc=A user access ID in the general form user:realmName/userUniqueId.
```
Comment (cjcarpen) [inline dev/com.ibm.ws.management.security/resources/OSGI-INF/l10n/metatype.properties]: What's meant by "general" form? What about: A user access ID in the format user:realmName/userUniqueId.

File: dev/com.ibm.ws.management.security/resources/OSGI-INF/l10n/metatype.properties [hunk 2]
```diff
@@ -21,5 +21,13 @@ role.administrator.desc=A collection of users and/or groups assigned the server
...
 role.user.desc=User assigned a role.
 
+role.userAccessId=User access ID
+role.userAccessId.desc=A user access ID in the general form user:realmName/userUniqueId. 
+
 role.group=Group name
 role.group.desc=Group assigned a role.
+
+role.groupAccessId=Group access ID
+role.groupAccessId.desc=A group access ID in the general form group:realmName/groupUniqueId.
```
Comment (cjcarpen) [inline dev/com.ibm.ws.management.security/resources/OSGI-INF/l10n/metatype.properties]: Same question as above.

File: dev/com.ibm.ws.management.security/resources/OSGI-INF/l10n/metatype.properties (hunk 1 above)
Comment (utle) [reply dev/com.ibm.ws.management.security/resources/OSGI-INF/l10n/metatype.properties]: The metatype property already deliver in the PR# https://github.com/OpenLiberty/open-liberty/pull/3627

File: dev/com.ibm.ws.management.security/resources/OSGI-INF/l10n/metatype.properties (hunk 2 above)
Comment (utle) [reply dev/com.ibm.ws.management.security/resources/OSGI-INF/l10n/metatype.properties]: Same above comments

**Comment by utle** [comment]
#build

**Pull request merged**